prime-router/src/test/kotlin/azure/BlobAccessTests.kt

```diff
@@ -448,6 +448,17 @@ class BlobAccessTests {
         assertThat(metadata.getBlobEndpoint()).isEqualTo("http://localhost:54321/devstoreaccount1/test")
     }
 
+    @Test
+    fun `test get blob endpoint when accountname and suffix are used`() {
+        val endpoint =
+            """
+                DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=keydevstoreaccount1;EndpointSuffix=core.windows.net
```

Comment:
I confirmed this shape by looking at what is the staging app configuration

Reply:
Only comment is perhaps name the vars `deployedEnvEndpoint` and `localEnvEndpoint` or something more descriptive.